vms/platformvm/txs/mempool/mempool.go

```diff
@@ -121,6 +124,15 @@ func New(
 	}, nil
 }
 
+func (m *mempool) Iterate(f func(tx *txs.Tx) bool) {
+	itr := m.unissuedTxs.NewIterator()
```

Comment:
I don't think it is safe to use an iterator of a linked hash map while the map could be arbitrarily changing.

Reply:
Added a lock to this mempool we acquire during iteration